efel/tests/featurenames.json

```diff
@@ -153,13 +159,5 @@
     "voltage_base",
     "voltage_deflection",
     "voltage_deflection_begin",
-    "voltage_deflection_vb_ssse",
```

Comment:
@anilbey , or is it because there are too many changes at the same time? Maybe you can change the order in a separate PR?

Reply:
But I already did that in a separate PR
https://github.com/BlueBrain/eFEL/pull/170